src/main/java/nugget/TaskList.java

```diff
@@ -0,0 +1,53 @@
+package nugget;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void addTask(Task task) {
+        tasks.add(task);
+    }
+
+    public void deleteTask(int index) {
+        tasks.remove(index);
+    }
+
+    public void markTask(int index) {
+        tasks.get(index).markTask();
+    }
+
+    public void unmarkTask(int index) {
+        tasks.get(index).unmarkTask();
+    }
+
+    public int size() {
```

Comment:
Great naming conventions. I think here size could be replace with something more verb like. Perhaps getSize().